api/graphql/resolvers/user.go

```diff
@@ -290,7 +295,6 @@ func (r *mutationResolver) UserRemoveRootAlbum(ctx context.Context, userID int,
 	}
 
 	var deletedAlbumIDs []int = nil
```

Comment:
It's better to remove this line, and creates the variable in line 323 as `deletedAlbumIDs, err := cleanup()`.

Reply:
The line 323 is inside the transaction. From the code logic I see that if the transaction fails, the function is terminated before the variable is used, however, will your proposal work (be compiled and accepted by linters) and is it ok from coding practice in Go (as I remember from Java that variables, used after code blocks, should be declared before these blocks, not inside them)?
If you confirm that this is ok for Go and will be accepted by the compiler and linters, I'll make this change. I just want to make sure)

Reply:
It's OK.

The transaction is not a part of the Go language but from the SQL engine. Only the result of `tx` is passed to the engine. Variables in the callback function are kept inside the process of the service binary.

Reply:
Unfortunately, it doesn't work: once I removed the declaration and modified the line 323 as proposed, here is what IDE shows to me
<img width="1028" alt="SCR-20241011-iauv" src="https://github.com/user-attachments/assets/857d070b-27eb-43d9-b6ae-134effb21ff8">

Reply:
Ah, OK. I did't notice the using of this variable is outside of the transaction.